### DISCUSSION TITLE
Translation of app

### DISCUSSION THREAD

**Discussion by kri164** [owntracks/android#1115] [Q&A] [answered]
Hi,
I did translate the app, but new beta does not include my translation.
Should I need something more to do than translating strings on poeditor.com?
Thank you

**Comment by growse** [comment, marked as answer]
Hi - apologies, this is probably a failure on my part. 

I can't quite figure out the right workflow to get update notifications from POEditor to then flow back into the repo. At the moment, it seems that I have to notice manually that the translations have been updated in POEditor, then manually push those into the repo. 

If anyone happens to know how to make this better / easier, please let me know.

In the meantime, I'll push some of the translations into the repo (I assume yours was the Czech translations?) and do another beta. Many thanks!
